### DISCUSSION TITLE
Big wow coming from a meteor lover

### DISCUSSION THREAD

**Discussion by chneau** [noam-honig/radweb#291] [General]
Starting with the desambiguity: meteor as in https://github.com/meteor/meteor 

I'm just half way through the video here https://www.youtube.com/watch?v=VN27pdF3a6c and I've been looking for soooooo long for an alternative to Meteor.

The real time with apply changes, the auth I will see later in the video. That's totally amazing. On top of that using vite for super fast development, I just am in love.

I wish I could attract more people knowing Meteor in here as I really think this is by far the best alternative I've found to Meteor in basically one library.

Something very similar I found before was this one: https://electric-sql.com/ 

I'm wondering if people here have suggestions about similar libraries that has the real time / auth magic "out of the box".

**Comment by jycouet** [comment]
I loved Meteor back in the days, nice to see you comparing to it.
Yeahhh, let's push this even more 💪

**Comment by yoni-rapoport** [comment]
I fully agree people should consider Remult as an alternative to Meteor that is less opinionated about stuff like build tools, frameworks. databases and real-time protocols, and more focused on simplifying everything around CRUD.

Thanks!
